website/source/docs/providers/aws/r/rds_cluster.html.markdown

```diff
@@ -79,6 +79,7 @@ Default: A 30-minute window selected at random from an 8-hour block of time per
      `false`. See [Amazon RDS Documentation for more information.](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Overview.DBInstance.Modifying.html)
 * `db_subnet_group_name` - (Optional) A DB subnet group to associate with this DB instance. **NOTE:** This must match the `db_subnet_group_name` specified on every [`aws_rds_cluster_instance`](/docs/providers/aws/r/rds_cluster_instance.html) in the cluster.
 * `db_cluster_parameter_group_name` - (Optional) A cluster parameter group to associate with the cluster.
+* `kms_key_id` - (Optional) The ARN for the KMS encryption key.
```

Comment:
we should mention that `storage_encrypted` needs to be set as well (I got an error otherwise). You can specify `storage_encrypted` without `kms_id`, but not the other way around it seems. Does that sound right?